src/basic/process-util.c

```diff
@@ -839,6 +839,62 @@ int pidref_get_start_time(const PidRef *pid, usec_t *ret) {
         return 0;
 }
 
+#define SIGCGT_MASK_BITS (sizeof(uint64_t) * CHAR_BIT)
```

Comment:
Maybe we already have some better way to do this in one of the util headers? I feel like we must do, but I couldn't immediately find it.